src/detector.c

```diff
@@ -354,11 +356,11 @@ void train_detector(char *datacfg, char *cfgfile, char *weightfile, int *gpus, i
             iter_map = iteration;
             mean_average_precision = validate_detector_map(datacfg, cfgfile, weightfile, 0.25, 0.5, 0, net.letter_box, &net_map);// &net_combined);
             printf("\n mean_average_precision (mAP@0.5) = %f \n", mean_average_precision);
-            if (mean_average_precision > best_map) {
+            if (mean_average_precision > 80.0) {
                 best_map = mean_average_precision;
-                printf("New best mAP!\n");
+                printf("mAP more than 80.0 %!\n");
                 char buff[256];
-                sprintf(buff, "%s/%s_best.weights", backup_directory, base);
+                sprintf(buff, "%s/%s_mark.weights", backup_directory, base);
```

Comment:
If we overwrite every time we get a >80% precision, we could potentially overwrite weight files with lower precisions (i.e. 90% could be overwritten with 85%).

Reply:
forgot adding iteration on the file name.
will be added soon.